### ISSUE TITLE
Units of MJy/sr not recognised

### ISSUE THREAD

**Issue by catrionamurray** [zkbt/chromatic#30]
When loading *x1dints.fits files with units of MJy/sr and (MJy/sr)^2, chromatic crashed as they're not standard astropy units. Can be bypassed using:

```
import astropy.units as u
MJy_sr = u.def_unit('MJy/sr', u.MJy/u.sr)
MJy_sr_sq = u.def_unit('(MJy/sr)^2', (u.MJy/u.sr)**2)
u.add_enabled_units([MJy_sr,MJy_sr_sq])
```

And then importing into chromatic

**Comment by zkbt** [comment]
Great catch, @catrionamurray ! It'd be great if we could please put this into `chromatic`, so other folks don't need to worry about it. I think the cleanest way to do that would be to:
- Move the unit-related imports from `chromatic/imports.py` into a separate `chromatic/units.py` file.
- Add your custom unit to there.
- Add `from .units import *` to the `chromatic/imports.py` file, so that we can then have access to all our same unit stuff, but including the new custom unit.
- Update the version number in `chromatic/version.py` to something higher than it currently is.

I think setting up the new `chromatic/units.py` file now is worthwhile because it seems likely we're going to encounter some other custom units we may want to add. 

When you have time in the next couple of weeks, could you please make those changes? You're a contributor to the repository, so you should be able to do that by:
- Cloning it to your computer.
- Making sure you have the most recent version of the `main` branch.
- Creating and checking out a new branch, based off `main`.
- Making the changes.
- Pushing your new branch up.
- Submitting a Pull Request from that branch. 

and then I'll take a look! Thanks!

**Comment by zkbt** [comment]
Closed by #40 . Yay, @catrionamurray !
